src/consumption.cpp

```diff
@@ -390,7 +390,7 @@ bool player::eat( item &food, bool force )
                                food.tname().c_str() );
     }
 
-    if( item::find_type( comest->tool )->is_tool() ) {
+    if( item::find_type( comest->tool )->tool ) {
```

Comment:
The `is_tool()` method nolonger exists. It was implemented as a virtual method overridden by `it_tool`